### ISSUE TITLE
Warns when calling p with arguments

### ISSUE THREAD

**Issue by ggl** [garu/Data-Printer#160]
Odd number of elements in hash assignment at /home/gelu/.local/lib/perl5/Data/Printer.pm line 81.

p ($arg1, $arg2);

**Comment by rabbiveesh** [comment]
the interface for p is documented as not taking more than one item to print.
The rest of the arguments are processed as key-value config options for
that particular print.

**Comment by garu** [comment]
Yup, what @rabbiveesh said. To print multiple variables with a single call, as documented throughout the manual and readme files, you have to group them together. So any of these work:

```perl
p [ $arg1, $arg2 ]->@*;                # postderef (perl 5.14 onwards)

p @{[ $arg1, $arg2 ]};                 # regular deref of anonymous ref (any version of perl 5)

&p( [ $arg1, $arg2 ] );                # ignoring signature + anonymous ref (ugly, but also works)

{ my @tmp = ($arg1, $arg2); p @tmp } ; # using a temporary array

p $arg1; p $arg2;                      # for 2 variables, this is also simple to do!
```

Remember: the second argument to `p()` is always a reference to customization options, like asking for a [fulldump](https://metacpan.org/pod/Data::Printer::Object#fulldump), loading a [filter](https://metacpan.org/pod/Data::Printer#Filters) or adding a [message](https://metacpan.org/pod/Data::Printer::Object#as) to help you find that particular print in your logs.
Hope this helps!
